### ISSUE TITLE
Advice for Dataloader when using a CNN LSTM Model

### ISSUE THREAD

**Issue by Christoph9402** [pytorch/pytorch#61812]
Hello,
I was wondering if anybody had advice for building a model, that can classify image sequences (binary classification). I want to use a webcam stream and for each frame I want to classfiy the video based on the last 5 frames.
I am thinking, a combination of a CNN and an LSTM would be good - the input for the LSTM are the extracted features from the CNN. 
I am having trouble understanding, how a dataloader should look like in this case. My data consists of multiple csv files and images. each csv file represents a sequence and contains the images of that sequence and the label. I should add, that each sequence has a different length. I want my model to have a structure like this :
<img width="1039" alt="Screenshot 2021-07-17 161414" src="https://user-images.githubusercontent.com/62546080/126040096-a58cdb81-e186-4f73-b581-b879ae716b8f.png">
Do you have any advice on how to build a dataloader? What do I have to do to make sure the shapes are correct? 
Thank you :)

cc @SsnL @VitalyFedyunin @ejguan @albanD @mruberry @jbschlosser

**Comment by albanD** [comment]
Hi,

We use github issues only for bugs or feature requests.
Please use the forum to ask questions: https://discuss.pytorch.org/

**Comment by jbschlosser** [comment]
Hey @Christoph9402! This sort of question is a better fit for the [PyTorch forums](https://discuss.pytorch.org/) - I'd recommend posting there for a better chance of getting useful insight.
